addons/account/static/tests/reconciliation_tests.js

```diff
@@ -208,9 +208,16 @@ var db = {
             if (!Datas.used.mv_lines[key]) {
                 throw new Error("Unknown parameters for get_move_lines_for_bank_statement_line: '"+ key + "'");
             }
-            return $.when(_.filter(Datas.used.mv_lines[key], function (line) {
-                return excluded_ids.indexOf(line.id) === -1 && (!partner_id || partner_id === line.partner_id);
-            }));
+            var lines = Datas.used.mv_lines[key]
+                .filter(function (line) {
+                    return excluded_ids.indexOf(line.id) === -1 && (!partner_id || partner_id === line.partner_id);
+                })
+                .map(function (line, i, src) {
+                    line.recs_count = src.length;
+                    return line;
+                })
+                .slice(0, options.params.limitMoveLines);
```

Comment:
are you sure options.params.limitMoveLines is defined? where is options defined?